osu.Game/Rulesets/Mods/Mod.cs

```diff
@@ -131,22 +131,31 @@ public virtual string SettingDescription
         /// </summary>
         public virtual Mod CreateCopy()
         {
-            var copy = (Mod)Activator.CreateInstance(GetType());
+            var result = (Mod)Activator.CreateInstance(GetType(), true);
+            result.CopyFrom(this);
+            return result;
+        }
+
+        /// <summary>
+        /// Copies properties of given mod into here, through changing value.
+        /// </summary>
+        /// <param name="them">The mod to copy properties from.</param>
+        public void CopyFrom(Mod them)
+        {
+            if (them.GetType() != GetType())
+                throw new ArgumentException($"Expected mod of type {GetType()}, got {them.GetType()}.", nameof(them));
 
-            // Copy bindable values across
             foreach (var (_, prop) in this.GetSettingsSourceProperties())
             {
-                var origBindable = prop.GetValue(this);
-                var copyBindable = prop.GetValue(copy);
+                var ourBindable = prop.GetValue(this);
+                var theirBindable = prop.GetValue(them);
 
                 // The bindables themselves are readonly, so the value must be transferred through the Bindable<T>.Value property.
-                var valueProperty = origBindable.GetType().GetProperty(nameof(Bindable<object>.Value), BindingFlags.Public | BindingFlags.Instance);
+                var valueProperty = theirBindable.GetType().GetProperty(nameof(Bindable<object>.Value), BindingFlags.Public | BindingFlags.Instance);
```

Comment:
Not sure again why this is a change. It probably doesn't make any functional difference, as any desynchronisation should result in catching fire, but from a readability standpoint alone `ourBindable` seems like a better choice for getting properties to copy into, as it is *this* instance that is being copied into. The type check at the start of the method should ensure there is nothing funny going on wrt types already.

Reply:
just a drive-by one, can revert.